apps/molgenis-components/src/components/constants.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  CODE_0: 48,
+  CODE_9: 57,
+  CODE_BACKSPACE: 8,
+  CODE_DELETE: 46,
+  CODE_MINUS: 45,
+  EMAIL_REGEX:
+    /^(([^<>()\\[\]\\.,;:\s@"]+(\.[^<>()\\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$|^$/,
+  HYPERLINK_REGEX:
+    /^((https?):\/\/)(www.)?[a-zA-Z0-9@:%._\\+~#?&//=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%._\\+~#?&//=]*)\/?$|^$/,
+  MAX_LONG: "9223372036854775807",
+  MIN_LONG: "-9223372036854775807",
+};
```

Comment:
no need for separate file is only used in one component ( if more use it , it's fine of course 0

Reply:
It is used in more, but they have not been moved yet. For now this file is a duplicate from the one in the styleguide, but since that will be removed I think this is fine for now.